src/components/fields/UnitCountShorthandField.js

```diff
@@ -56,9 +57,6 @@ export default class UnitCountShorthandField extends React.Component {
 		let formData = this.props.formData;
 		const {shorthandField, pairCountField} = getUiOptions(this.props.uiSchema);
 
-		let timestamp = Date.now();
-		this.promiseTimestamp = timestamp;
```

Comment:
Can you elaborate why these were removed? Now if the response for the last sent request arrives sooner than some previous one, the previous response will overwrite the last request. Some debounce mechanic would be optimal here, but that's outside of this PR's scope.